aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/src/main/resources/static/App/Scripts/todoListSvc.js

```diff
@@ -9,10 +9,10 @@ angular.module('todoApp')
                 return $http.get('/api/todolist/' + id);
             },
             postItem: function (item) {
-                return $http.post('/api/todolist/', item);
+                return $http.post('/api/todolist', item);
             },
             putItem: function (item) {
-                return $http.put('/api/todolist/', item);
+                return $http.put('/api/todolist', item);
```

Comment:
What will happen if keep original code?

Reply:
Currently, it's OK to run this sample, but not accurate, see more from https://github.com/Azure-Samples/azure-spring-boot-samples/pull/582#issue-1460923272